src/views/SystemIntake/ContractDetails/index.tsx

```diff
@@ -123,7 +123,10 @@ const ContractDetails = ({ systemIntake }: ContractDetailsProps) => {
       },
       costs: values.costs,
       contract: {
-        ...values.contract,
+        // ...values.contract,
+        hasContract: values.contract.hasContract,
+        contractor: values.contract.contractor,
+        number: values.contract.vehicle, // TODO - EASI-1977 - horrible hack to get this working with updated GQL schema
```

Comment:
@adamodd I'm not sure how it happened, but your fix to my hacky code got overwritten somehow. Can you re-fix this?

Reply:
Fixed in https://github.com/CMSgov/easi-app/pull/1755/commits/18fe494adde68ba027abab98a57908e320cfe9d7.